src/tools/compiletest/tidy-rustdoc.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+indir="${1:?Missing argument 1: input directory}"
+
+tidy () {
+  {
+    # new-inline-tags is workaround for:
+    #   https://github.com/rust-lang/stdarch/issues/945
+    #   https://github.com/rust-lang/mdBook/issues/1372
+    command tidy \
+        --indent yes \
+        --indent-spaces 2 \
+        --wrap 0 \
+        --show-warnings no \
+        --markup yes \
+        --quiet yes \
+        --new-inline-tags 'c t' \
```

Comment:
Probably don't need these (and the comment) since you're not checking the book and `stdarch` docs.